### PR TITLE
[SEM-007] #10 : Secondary Button Icon +  

### PR DESCRIPTION
**Pull request by 11royer**
Como desarrollador, necesito crear y probar un componente de icono +. A partir del Icono (+) de primary button agregue las variaciones de color.

### CODE REVIEW

**Review by Ditmar — CHANGES_REQUESTED** [review]
The rebase was not done correctly.

File: package-lock.json
[the remark this answers is not in the document]
Comment (11royer) [reply package-lock.json:1]: now fixed

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@testing-library/dom": "10.4.0",
     "@testing-library/jest-dom": "6.4.5",
     "@testing-library/react": "16.0.1",
+    "@testing-library/user-event": "^14.5.2",
```
[the remark this answers is not in the document]
Comment (11royer) [reply package.json]: The purpose of the library is to simulate hover and focus on the button, which allows interaction with the mouse:
await userEvent.hover(button);
await userEvent.unhover(button);

As well as interaction on focus:
await userEvent.tab();

**Comment by 11royer** [comment]
https://github.com/Ditmar/backoffice-openscience/pull/76

**Comment by 11royer** [comment]
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-007/?path=src/ui/components/atoms/secondary-button-icon

**Pull request closed without merging**